### DISCUSSION TITLE
How to clear failed export tasks

### DISCUSSION THREAD

**Discussion by flamevip** [outline/outline#6152] [Self Hosting] [answered]
![20231114154109](https://github.com/outline/outline/assets/41901255/951540fa-584c-4440-8d42-ab1b1e7df0a7)

**Comment by tommoor** [comment, marked as answer]
Running scheduled tasks will clear this list, see:

https://docs.getoutline.com/s/hosting/doc/scheduled-jobs-RhZzCt770H

**Reply by flamevip** [reply]
Thank you very much for your answer. Actually, I don't want to end the queue of unfinished tasks. I want to delete all the failed tasks here. obsessive-compulsive disorder
